csbdeep/internals/blocks.py

```diff
@@ -174,3 +174,99 @@ def f(inp):
         return x
 
     return f
+
+
+
+def fpn_block(n_depth=3,
+              n_filter_base=64,
```

Comment:
Why `n_filter_base` default much larger as compared to `unet_block`?

Reply:
We could make `unet_block` default larger too...

Reply:
I'd rather not for backwards compatibility.